### DISCUSSION TITLE
Octane prevents to load styles

### DISCUSSION THREAD

**Discussion by s-fog** [laravel/nova-issues#4564] [Q&A] [answered]
- Laravel Version: ^9
- Nova Version: ^4
- PHP Version: 8.1
- Database Driver & Version: mysql8
- Operating System and Version: ubuntu 20.04
- Browser type and version: any

### Description:
When i activate octane it disables custom css or js to load.
Inside NovaServiceProvider is
`public function boot()
    {
        Nova::style('admin', asset('/css/custom_nova.css'));
        parent::boot();
    }`
When i load admin panel without octane styles are inside head, as expected. If i load with octane, there is no this styles at all. 

Can u help to fix this please?

**Comment by crynobone** [comment, marked as answer]
Wrap `Nova::style()` inside `Nova::serving()`
